apps/api-reference/src/components/EvmApi/networks.ts

```diff
@@ -0,0 +1,18 @@
+import { arbitrum, avalanche, mainnet, sepolia } from "wagmi/chains";
+
+export const getContractAddress = (networkId: number) =>
+  isSupportedNetwork(networkId)
+    ? NETWORK_TO_CONTRACT_ADDRESS[networkId]
+    : undefined;
+
+const NETWORK_TO_CONTRACT_ADDRESS = {
+  [mainnet.id]: "0x4305FB66699C3B2702D4d05CF36551390A4c69C6",
+  [avalanche.id]: "0x4305FB66699C3B2702D4d05CF36551390A4c69C6",
+  [arbitrum.id]: "0xff1a0f4744e8582DF1aE09D5611b887B6a12925C",
+  [sepolia.id]: "0xDd24F84d36BF92C65F92307595335bdFab5Bbd21",
```

Comment:
sepolia is not a network in its own, it usually refers to testnet, for e.g. eth sepolia, arbitrum sepolia, base sepolia are all evms but they have different contract addresses so we shouldnt assume that they are all deployed on the same address

Reply:
"sepolia" here comes out of wagmi/chains which itself re-exports viem's chains definitions which uses `sepolia` as the name for Ethereum sepolia: https://github.com/wevm/viem/blob/main/src/chains/definitions/sepolia.ts

Here I'm not naming it "sepolia", I'm just using the definition out of viem.

I can rename the import declaration though to make it a bit clearer if you think that would help.

Reply:
Also note this object doesn't match a network _name_ to a contract address, it maps a network _id_ to the contract address.  The names aren't really important, they're just local variables.

Reply:
We will be replacing this `contract-manager` once it gets published.